plotter/boxplot.go

```diff
@@ -287,19 +312,11 @@ func (o boxPlotOutsideLabels) Label(i int) string {
 	return o.labels[i]
 }
 
-// HorizBoxPlot is like a regular BoxPlot, however,
+// horizBoxPlot is like a regular BoxPlot, however,
 // it draws horizontally instead of Vertically.
-type HorizBoxPlot struct{ *BoxPlot }
-
-// MakeHorizBoxPlot returns a HorizBoxPlot,
-// plotting the values in a horizontal box plot
-// centered along a fixed location of the y axis.
-func MakeHorizBoxPlot(w vg.Length, loc float64, vs Valuer) (HorizBoxPlot, error) {
-	b, err := NewBoxPlot(w, loc, vs)
-	return HorizBoxPlot{b}, err
-}
+type horizBoxPlot struct{ *BoxPlot }
```

Comment:
Could you please add a TODO note that we should just merge this code into the methods above, like you've done for bar charts?